examples/apps/ai-collab/src/components/TaskCard.tsx

```diff
@@ -288,46 +289,43 @@ export function TaskCard(props: {
 				</Popover>
 			)}
 
-			<Stack direction="row" sx={{ width: "100%" }} spacing={2}>
-				<Stack sx={{ flexGrow: 1, direction: "row" }}>
-					<Stack direction="row">
-						<TextField
-							id="input-description-label-id"
-							label="Description"
-							value={task.description}
-							onChange={(e) => (props.sharedTreeTask.description = e.target.value)}
-							sx={{ height: "100%", width: "100%" }}
-							slotProps={{
-								input: {
-									multiline: true,
-									sx: {
-										alignItems: "flex-start",
-										backgroundColor:
-											fieldDifferences.changes.description === undefined ? "white" : "#a4dbfc",
-									},
-								},
-								inputLabel: {
-									sx: { fontWeight: "bold" },
-								},
-							}}
-						/>
-						{fieldDifferences.changes.description !== undefined && (
-							<IconButton
-								onClick={(event) => {
-									setDiffOldValue(
-										fieldDifferences.changes.description
-											?.oldValue as SetStateAction<ReactNode>,
-									);
-									setDiffOldValuePopoverAnchor(event.currentTarget);
-								}}
-							>
-								<Icon icon="clarity:info-standard-line" width={20} height={20} />
-							</IconButton>
-						)}
-					</Stack>
-				</Stack>
+			<Stack direction="row" sx={{ mb: 2 }}>
```

Comment:
Not necessarily for this PR, but there is a lot going on in this component - might be good to split it up into smaller ones.